src/Components/ShippingMethod/class-shipping-method-helper.php

```diff
@@ -20,34 +22,78 @@
  */
 class Shipping_Method_Helper {
 
+	const SHIPPING_ID    = '_packlink_shipping_method_id';
+	const DROP_OFF_ID    = '_packlink_drop_off_point_id';
+	const DROP_OFF_EXTRA = '_packlink_drop_off_extra';
+
 	/**
-	 * Returns path to carrier logo or empty string if that logo file doesn't exist.
+	 * Returns Packlink shipping method.
+	 *
+	 * @param \WC_Order $wc_order WooCommerce order.
 	 *
-	 * @param string $carrier_name Name of the carrier.
+	 * @return ShippingMethod Returns Packlink shipping method.
 	 *
-	 * @return string Carrier image url.
+	 * @throws \Logeecom\Infrastructure\ORM\Exceptions\QueryFilterInvalidParamException
+	 * @throws \Logeecom\Infrastructure\ORM\Exceptions\RepositoryNotRegisteredException
 	 */
-	public static function get_carrier_logo( $carrier_name ) {
-		$file_path = dirname( dirname( __DIR__ ) ) . '/resources/images/carriers/';
-		$base_path = Shop_Helper::get_plugin_base_url() . 'resources/images/carriers/';
-		$default   = Shop_Helper::get_plugin_base_url() . 'resources/images/box.svg';
+	public static function get_packlink_shipping_method_from_order( \WC_Order $wc_order ) {
+		$shipping = $wc_order->get_shipping_methods();
+		if ( empty( $shipping ) ) {
+			return null;
+		}
 
+		$shipping_method_id = - 1;
+		foreach ( $shipping as $item_id => $shipping_item ) {
```

Comment:
Is it `$shipping` an array? 
If doesn't matter from which `$shipping_item` it gets the data, may it gets the data from the first or last element instead of iterate the whole array?

Reply:
I agree. Corrected.